site/content/docs/5.0/components/tooltips.md

```diff
@@ -261,7 +261,7 @@ Note that for security reasons the `sanitize`, `sanitizeFn`, and `allowList` opt
     <tr>
       <td><code>fallbackPlacements</code></td>
       <td>array</td>
-      <td><code>['top', 'right', 'bottom', 'left']</code></td>
+      <td><code>['top', 'bottom', 'left', 'right']</code></td>
```

Comment:
Ah, still not reverted :smile:

Reply:
whoops - forgot to stage that file... incoming...